src/settings/Validation/engine/handlers.js

```diff
@@ -61,3 +60,15 @@ export const hasPositivereplacementAllowed = (value, model) => {
 
   return replacementAllowed === 'true';
 };
+
+export const hasPatronBilledAfterAgedToLostValue = (value, model) => {
+  const patronBilledAfterAgedToLostValue = get(model, 'patronBilledAfterAgedLost.duration');
+
+  return (isNotEmpty(patronBilledAfterAgedToLostValue) && isNotEmpty(value)) || !isNotEmpty(patronBilledAfterAgedToLostValue);
+};
+
+export const hasChargeAmountItemSystemSelected = (value, model) => {
+  const chargeAmountItemSystem = get(model, 'chargeAmountItemSystem');
+
+  return (chargeAmountItemSystem === 'true' && isNotEmpty(value)) || chargeAmountItemSystem !== 'true';
```

Comment:
We really want to handle these boolean values as strings, i.e. `'true'` not `true`?

Reply:
@zburke Field component sends its value in string, even if default was set as boolean. Parse option for Field will store boolean in redux store, but validator will receive string representation. Lost item fee policy was implemented by another team, so I am just following conventions they did.

Reply:
@zburke done